consoles/virtio_terminal.pm

```diff
@@ -110,33 +108,38 @@ sub load_snapshot {
 
   open_socket();
 
-Opens a socket to the host end of the virtio_console. This need to be
-corresponding to the console which was created by qemu-backend.
+Opens a the read and write pipe based on C<$pipe_prefix>.
 
-Returns the file descriptor for the open socket, otherwise it dies.
+Returns the read and write file descriptors for the open sockets,
+otherwise it dies.
 
 =cut
 sub open_socket {
```

Comment:
This should be changed to pipe. Maybe in the future we will allow connecting to a remote QEMU instance.